src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -540,13 +512,20 @@ function IOURequestStepConfirmation({
                             // but not all of them (maybe someone skipped out on dinner). Then it's nice to be able to select/deselect people from the group chat bill
                             // split rather than forcing the user to create a new group, just for that expense. The reportID is empty, when the action was initiated from
                             // the floating-action-button (since it is something that exists outside the context of a report).
-                            canModifyParticipants={!transaction.isFromGlobalCreate}
-                            policyID={report.policyID}
+                            canModifyParticipants={!transaction?.isFromGlobalCreate}
+                            policyID={report?.policyID}
                             bankAccountRoute={ReportUtils.getBankAccountRoute(report)}
-                            iouMerchant={transaction.merchant}
-                            iouCreated={transaction.created}
+                            iouMerchant={transaction?.merchant}
+                            iouCreated={transaction?.created}
                             isDistanceRequest={requestType === CONST.IOU.REQUEST_TYPE.DISTANCE}
                             shouldShowSmartScanFields={requestType !== CONST.IOU.REQUEST_TYPE.SCAN}
+                            reportActionID={undefined}
+                            hasSmartScanFailed={undefined}
+                            isEditingSplitBill={undefined}
+                            isReadOnly={undefined}
+                            isScanRequest={undefined}
+                            listStyles={undefined}
+                            payeePersonalDetails={undefined}
```

Comment:
Why do we need these updates? Maybe it's better add `ts-expect-error` till `MoneyRequestConfirmationList` is migrated to TS?